### PR TITLE
Adds first repo for lib updates (many more to come).

### PR DESCRIPTION
**Pull request by codingjeremy**
I wanted to separate all android samples so they don't pollute the top level directory (like firebase did).

I believe I still need the common.cfg from review the firebase sample. I will add more .cfg files for other repos once this one looks good. 

Let me know if you have any questions, thanks!

### CODE REVIEW

**Comment by codingjeremy** [comment]
One other note, I sent an invite to DPEBot for this org but it has not accepted.

**Comment by tswast** [comment]
I'll manually approve the invite. Could be our webhook is down. The Yoshi team moved to a different infrastructure for their bots, so we haven't really paid much attention to that piece of dpebot. The dependency updates still work, though.

File: .kokoro/android/common.cfg [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/dpebot"
+
+# Common env vars for all repositories and builds.
+env_vars: {
+    key: "DPEBOT_GITHUB_TOKEN_FILE"
+    value: "github-token.txt"
+}
+env_vars: {
+    key: "DPEBOT_GIT_USER_NAME"
+    value: "DPE bot"
```
Comment (samtstern) [inline .kokoro/android/common.cfg]: Is this correct?

Comment (samtstern) [reply .kokoro/android/common.cfg]: Oh it works for Firebase so ... guess it doesn't matter.

Comment (codingjeremy) [reply .kokoro/android/common.cfg]: I copied the [firebase common](https://github.com/GoogleCloudPlatform/repository-gardener/blob/master/.kokoro/firebase/common.cfg)... maybe it isn't used?

**Pull request merged**